docs/pages/setup/reference/cli.mdx

```diff
@@ -1289,6 +1289,68 @@ Print cluster version:
 tctl version
 ```
 
+## Resource Filtering
```

Comment:
Thanks a lot for putting this together! There are a lot of guides that hint at our filter syntax, but nothing so far that spells it out in a comprehensive reference.

Since our filter predicate syntax is used throughout Teleport, though, I do think the `### Predicate Language` section needs to be a separate reference guide in `docs/pages/setup/reference`. We can then link to that new page in this H2 section.

Do different parts of the predicate syntax apply to different situations in Teleport? I don't see the `contains` function in this reference, for example. If this is the case, we should separate the predicate reference guide into subsections that apply to these different situations.

I think a central, dedicated predicate reference will go a long way toward clarifying this syntax, which so far is only documented via examples here and there in our docs.

Reply:
created separate page for predicate langugae: https://github.com/gravitational/teleport/blob/lisa/update-cli-doc/docs/pages/setup/reference/predicate-language.mdx